### ISSUE TITLE
cargo build fails without cmake installed

### ISSUE THREAD

**Issue by aig787** [bike-barn/hermit#138]
When I first tried to compile this I got 

```
error: failed to run custom build command for `libgit2-sys v0.3.12`
process didn't exit successfully: `/Users/aarongriffin/hermit/target/debug/build/libgit2-sys-e75bed5ab82a71d5/build-script-build` (exit code: 101)
--- stdout
running: "cmake" "/Users/aarongriffin/.cargo/registry/src/github.com-1ecc6299db9ec823/libgit2-sys-0.3.12/libgit2" "-DBUILD_SHARED_LIBS=OFF" "-DBUILD_CLAR=OFF" "-DCURL=OFF" "-DCMAKE_INSTALL_PREFIX=/Users/aarongriffin/hermit/target/debug/build/libgit2-sys-e75bed5ab82a71d5/out" "-DCMAKE_C_FLAGS= -O0 -ffunction-sections -fdata-sections -m64 -fPIC" "-DCMAKE_C_COMPILER=/usr/bin/cc" "-DCMAKE_CXX_FLAGS= -O0 -ffunction-sections -fdata-sections -m64 -fPIC" "-DCMAKE_CXX_COMPILER=/usr/bin/c++" "-DCMAKE_BUILD_TYPE=Debug"

--- stderr
thread 'main' panicked at '
failed to execute command: No such file or directory (os error 2)
is `cmake` not installed?
```

Builds fine with cmake installed. Should we add that as a dependency in the readme?

**Comment by cgrinker** [comment]
PR above should be accepted and this ticket should be closed.

**Comment by reillysiemens** [comment]
Agreed. #139 should be merged and #138 should be closed.
